lib/routes/participant.js

```diff
@@ -0,0 +1,24 @@
+const router = require('koa-router')()
+const models = require('../models')
+/* Participant */
+router
+  .get('/participant/:id', async (ctx) => {
+    let fbid = ctx.params.id
+    ctx.body = await models.db.participants.findOne({where: {fbid: fbid}})
```

Comment:
@moizjv suggested there is a shorthand for `{where: {fbid: fbid}}` => `{where: {fbid}}`